src/code/z_lib.c

```diff
@@ -590,16 +590,26 @@ void Color_RGBA8_Copy(Color_RGBA8* dst, Color_RGBA8* src) {
     dst->a = src->a;
 }
 
-void func_80078884(u16 sfxId) {
+/**
+ * Play a sound effect at the center of the screen.
+ */
+void Lib_PlaySfx(u16 sfxId) {
     Audio_PlaySfxGeneral(sfxId, &gSfxDefaultPos, 4, &gSfxDefaultFreqAndVolScale, &gSfxDefaultFreqAndVolScale,
                          &gSfxDefaultReverb);
 }
 
-void func_800788CC(u16 sfxId) {
+/**
+ * Play a sound effect at the center of the screen. Identical to `Lib_PlaySfx`.
+ */
+void Lib_PlaySfx_Centered(u16 sfxId) {
```

Comment:
With the two being identical, why is one named "Centered" and the other not?
(I feel like you gave an explanation somewhere already but I can't remember where, sorry)

Reply:
The rough idea I had was that `Lib_PlaySfx` (or `Sfx_PlaySfx`) was generic enough that it could be assumed that the default position was used i.e. center of the screen, and that adding `_Centered` was just redundant. But using that redundency, I could name the two identical functions `Lib_PlaySfx` and `Lib_PlaySfx_Centered` with slightly more interesting names than just `Lib_PlaySfx` and `Lib_PlaySfx2`.

I ended up just going with the 2 notation though to avoid confusion, and stuck with the `Centered` as part of the name for both sfx functions